frontend/dockerfile/release/releases.json

```diff
@@ -0,0 +1,32 @@
+{
+  "0.10.0": "1.4.0",
```

Comment:
I'd rather avoid this. It is messy to manage. If someone needs this information it can be implied from the buildkit revision. Users should not try to detect the default version but put a fixed version on top of the Dockerfile.

Reply:
Sure will remove this commit